src/components/ComponentPages/TopicDetails/NewsFeedsCard/index.tsx

```diff
@@ -151,7 +153,7 @@ const NewsFeedsCard = ({ newsFeed }) => {
 
                       {editNews && (
                         <Tooltip
-                          title={news.owner_flag ? "" : "must owner to edit"}
+                          title={news.owner_flag ? "" : "must be owner to edit"}
```

Comment:
Message must must come from the common file

Reply:
Done